src/check_impl.h

```diff
@@ -65,6 +65,8 @@ struct TCase
     List *unch_tflst;
     List *ch_sflst;
     List *ch_tflst;
+    unsigned int num_tags;
+    char **tags;
```

Comment:
OK - will take a look at the list stuff - wanted to get the self tests all passing reliably first - but touch wood that seems to have happened.